src/msw/menuitem.cpp

```diff
@@ -366,12 +366,12 @@ void MenuDrawData::Init()
         const NONCLIENTMETRICS& metrics = wxMSWImpl::GetNonClientMetrics();
 
         CheckMargin.cxLeftWidth =
-        CheckMargin.cxRightWidth  = ::GetSystemMetrics(SM_CXEDGE);
+        CheckMargin.cxRightWidth  = wxGetSystemMetrics(SM_CXEDGE);
```

Comment:
We really need to get the actual window here somehow (i.e. `wxMenu::GetWindow()`) because using the wrong one here could easily result in drawing in the wrong size.

Reply:
That's not possible here, as this class (`MenuDrawData`) is not associated with a menu or window. It should be taken care of in the pm-dpi PR.